build_docs.pl

```diff
@@ -337,9 +337,13 @@ sub check_links {
         say $link_checker->report;
```

Comment:
I'm really confused how the logic in this method works. It seems like "has_bad" returns "true" if there _aren't_ any bad links? 😕 . In which case we could maybe combine the logic for `if ( $link_checker->has_bad || $Opts->{warnlinkcheck})`?

Reply:
Yeah, that seems like a misnomer to me too. I considered renaming it as part of this PR, but was concerned what other things would crawl out from under the rock if I did so

Reply:
I've implemented your suggestion in ba5bd2165ac4badafd0797b8de9bb89ca0109849